bootstrap/000_setup_base_tools.sql

```diff
@@ -16,6 +16,23 @@ BEGIN
       INSERT (key, value) VALUES (source.key, source.value);
 END;
 
+CREATE OR REPLACE PROCEDURE internal.get_config(key string) RETURNS STRING LANGUAGE SQL
+    AS
+DECLARE
+    EXCEPTION_DUP_KEYS EXCEPTION(-20001, 'More than 1 rows in table internal.config for the specified key');
```

Comment:
wouldnt it be easier to use a unique constraint on the key? That would simplify this method considerably

Reply:
good idea.